br/pkg/lightning/backend/external/writer.go

```diff
@@ -343,8 +343,8 @@ func (w *Writer) Close(ctx context.Context) error {
 
 	logutil.Logger(ctx).Info("close writer",
 		zap.String("writerID", w.writerID),
-		zap.String("minKey", hex.EncodeToString(w.minKey)),
-		zap.String("maxKey", hex.EncodeToString(w.maxKey)))
+		zap.String("startKey", hex.EncodeToString(w.minKey)),
+		zap.String("endKey", hex.EncodeToString(w.maxKey)))
```

Comment:
Is this change expected? `endKey` -> `w.maxKey`

Reply:
https://github.com/pingcap/tidb/pull/47648/commits/a4e74e7084e8cf103bffc2c1d6a8a7cb58d7c64e